packages/target-truffle-v5-test/types/truffle-contracts/types.d.ts

```diff
@@ -62,6 +62,8 @@ declare namespace Truffle {
     from?: string;
     gas?: BN | number | string;
     gasPrice?: BN | number | string;
+    maxPriorityFeePerGas?: string | number | BN;
+    maxFeePerGas?: string | number | BN;
```

Comment:
```suggestion
    maxFeePerGas?: BN | number | string;
```
Same order like others.